Netherlands (nl-NL)/Oqtane.Client/Oqtane.Client.nl-NL.cmd

```diff
@@ -1,5 +1,5 @@
 dotnet build -c Release ..\..\Oqtane.sln
-mkdir ..\..\Oqtane.Server\bin\Debug\net5.0\nl-NL
-copy ..\..\Oqtane.Server\bin\Release\net5.0\nl-NL\Oqtane.Client.resources.dll ..\..\Oqtane.Server\bin\Debug\net5.0\nl-NL\Oqtane.Client.resources.dll
+mkdir ..\..\Oqtane.Server\bin\Debug\net6.0\nl-NL
```

Comment:
@sbwalker do we track TFM changes for each release or just we overwrite the old one?

Reply:
@hishamco what does TFM stand for? I am not sure how to answer your question... basically Oqtane now has full support for localization which can be installed as nuget packages at runtime - but the only person who seems to be maintaining their resource files for each release is @leigh-pointer... so I guess my answer is that I would defer to him for his opinion on this topic.

Reply:
TFM stands for Target Framework Monkers, regarding the maintanance for Arabic, I just stopped for awhile because I don't have a good internet connection, also I saw a lot of changes in Oqtane as well as localization, I don't know what's the plan or the roadmap, because there's no roadmap or features shared with the community to work with. I remember I asked this in on of the issues or discussion but there's no respond

I hope to share the roadmap and the upcoming features with the community to make the project LIVE and community can engage as much as they can

All the best

Reply:
@sbwalker does the framework handle what it installs? As in, i run version 2.3.1 but i try to install language pack 3.0.0, if  3.0.0 is installed then the translation wont fully work! TMF and dependancies handles that. 

Currently we have no target in the nuget package, and it builds with a warning. To be honest, I’m not sure if was like this before but i have been very observant with the builds so maybe this is new! I will explore.